tests/agent/concurrent_requests.py

```diff
@@ -105,7 +105,7 @@ def load_test(self):
         self.logger.debug("Starting tornado I/O loop")
         ioloop.IOLoop.instance().start()
 
-    def check_counts(self, it, max_wait=60, backoff=1):
+    def check_counts(self, it, max_wait=60, backoff=5):
```

Comment:
@kuisathaverat You only increased the backoff here, not the wait time. I also looked at this but talked to Silvia and got to the conclusion it is not likely. Also, there is always exactly one missing when they fail, which doesn't seem to fit.

Reply:
I know, I want to not bother Elasticsearch, to see if issues are related to Elasticsearch load, seems not.